libopae/src/bitstream_int.h

```diff
@@ -142,6 +142,19 @@ fpga_result validate_bitstream_metadata(fpga_handle handle,
 fpga_result read_gbs_metadata(const uint8_t *bitstream,
 			      struct gbs_metadata *gbs_metadata);
 
+/**
+* Reads interface id high and low values
+*
+* Reads interface id from sysfs.
+*
+* @param[in] handle   FME handle
+* @param[out] id_l    Interface id low
+* @param[out] id_h    Interface id lHigh
+* @returns             FPGA_OK on success
+*/
+fpga_result get_interface_id(fpga_handle handle,
```

Comment:
Querying the FME's guid property actually returns the PR interface ID, so this method of obtaining it isn't strictly necessary.  That's an optimization that we can make later, though.

Reply:
I agree with @tswhison , we can (and should do this later). You should either add a TODO here or track it elsewhere.